server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/MIMEMessageConverterTest.java

```diff
@@ -420,4 +420,50 @@ public void convertToMimeShouldAddAttachmentAndMultipartAlternativeWhenOneAttach
         assertThat(attachmentPart.getMimeType()).isEqualTo(expectedMimeType);
         assertThat(attachmentPart.getHeader().getField("Content-ID").getBody()).isEqualTo(expectedCID);
     }
-}
\ No newline at end of file
+
+    @Test
+    public void convertToMimeShouldAddAttachmentAndContainsIndicationAboutTheWayToEncodeFilenamesAttachmentInTheInputStreamWhenSending() {
```

Comment:
I guess we are only interested about the fileName... Maybe some assertions are not needed here....